Makefile

```diff
@@ -583,16 +579,10 @@ run-ci: build-ci
 	$(MAKE) test-ci -j1
 
 .PHONY: test-release
-test-release: test-build
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER)
-
 .PHONY: test-debug
-test-debug: test-build
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=debug
-
-.PHONY: test-message
-test-message: test-build
```

Comment:
This target and the ones below that are being removed are also present in vcbuild.bat. Might be worth removing these from there too?